web/src/stream_ui_updates.ts

```diff
@@ -590,7 +590,9 @@ export function update_stream_privacy_choices(policy: string): void {
     if (!overlays.streams_open()) {
         return;
     }
+    // eslint-disable-next-line no-jquery/no-sizzle
     const stream_edit_panel_opened = $("#stream_permission_settings").is(":visible");
+    // eslint-disable-next-line no-jquery/no-sizzle
     const stream_creation_form_opened = $("#stream-creation").is(":visible");
```

Comment:
This is very broken in multiple ways. `#stream_permission_settings` is an ID that’s incorrectly reused (once within `#stream-creation`, once within `#stream_settings`). The only way to end up with `$container = $("#stream-creation")` below is to have a different tab selected, which doesn’t make sense as a thing to check for here.

Reply:
@sahil839 can you investigate this as a follow-up to this PR?

Reply:
Opened #34674.